emails.coffee

```diff
@@ -57,9 +57,15 @@ Helpers =
   # creates an absolute URL.
   #
   #     {{ emailUrlFor 'myRoute' param='foo' }} => http://root-domain.com/my-route/foo
-  emailUrlFor: (route, params) ->
-    if Router
-      Utils.joinUrl Mailer.settings.baseUrl, Router.path.call(Router, route, params.hash)
+  emailUrlFor: (routeName, params) ->
+    # if Iron Router
+    if Router?
+      Utils.joinUrl Mailer.settings.baseUrl, Router.path.call(Router, routeName, params.hash)
+
+    # if Flow Router
+    if FlowRouter?
+      baseUrl = Utils.joinUrl Mailer.settings.baseUrl, FlowRouter.path.call(FlowRouter, routeName, params.hash)
```

Comment:
ai, that line should be.

```
 # if Flow Router
if FlowRouter?
   Utils.joinUrl Mailer.settings.baseUrl, FlowRouter.path.call(FlowRouter, routeName, params.hash)
```

I'll change that. Sure, we can add a fallback mechanism